bigquery-flash-data-sync/internal/config/config.go

```diff
@@ -0,0 +1,323 @@
+
+	tables, err := loadTableConfigs(logger, dbID)
+	if err != nil {
+		return nil, fmt.Errorf("failed to load table configs: %w", err)
+	}
+
+	return &model.DatabaseConfig{
+		Name:             dbID,
+		Type:             dbType,
+		Host:             host,
+		Port:             port,
+		DatabaseName:     database,
+		User:             user,
+		ConnectionString: connString,
+		Tables:           tables,
+		Enabled:          enabled,
+	}, nil
+}
+
+// loadTableConfigs loads table configurations for a specific database.
+// Tables are specified via {PREFIX}_TABLES environment variable.
+func loadTableConfigs(logger *zap.Logger, dbID string) (map[string]*model.TableConfig, error) {
+	prefix := strings.ToUpper(dbID) + "_"
+	tablesStr := getEnv(prefix+"TABLES", "")
+
+	if tablesStr == "" {
+		return nil, fmt.Errorf("%sTABLES is required (comma-separated list of table names)", prefix)
+	}
+
+	tables := make(map[string]*model.TableConfig)
+	tableList := strings.Split(tablesStr, ",")
+
+	for _, tableName := range tableList {
+		tableName = strings.TrimSpace(tableName)
+		if tableName == "" {
+			continue
+		}
+
+		tableConfig := loadTableConfig(logger, dbID, tableName)
+		tables[tableName] = tableConfig
+	}
+
+	if len(tables) == 0 {
+		return nil, fmt.Errorf("no valid tables found for database '%s'", dbID)
+	}
+
+	return tables, nil
+}
+
+// loadTableConfig loads configuration for a specific table.
+// Environment variables are prefixed with {DB_ID}_{TABLE_NAME}_ in uppercase.
+func loadTableConfig(logger *zap.Logger, dbID, tableName string) *model.TableConfig {
+	prefix := strings.ToUpper(dbID) + "_" + strings.ToUpper(tableName) + "_"
+
+	targetTable := getEnv(prefix+"TARGET_TABLE", tableName)
+	primaryKey := getEnv(prefix+"PRIMARY_KEY", "id")
+	timestampCol := getEnv(prefix+"TIMESTAMP_COLUMN", "")
+	columnsStr := getEnv(prefix+"COLUMNS", "")
+	batchSize := parseInt(logger, prefix+"BATCH_SIZE", "0", 0)
+	enabled := parseBool(getEnv(prefix+"ENABLED", "true"))
+
+	var columns []string
+	if columnsStr != "" {
+		for _, col := range strings.Split(columnsStr, ",") {
+			col = strings.TrimSpace(col)
+			if col != "" {
+				columns = append(columns, col)
+			}
+		}
+	}
+
+	return &model.TableConfig{
+		Name:            tableName,
+		TargetTable:     targetTable,
+		PrimaryKey:      primaryKey,
+		TimestampColumn: timestampCol,
+		Columns:         columns,
+		BatchSize:       batchSize,
+		Enabled:         enabled,
+	}
+}
+
+// buildConnectionString creates a database connection string based on type
+func buildConnectionString(dbType, host, port, database, user, password, prefix string) string {
+	connTimeout := getEnv(prefix+"DB_CONN_TIMEOUT", "30s")
+	readTimeout := getEnv(prefix+"DB_READ_TIMEOUT", "60s")
+	writeTimeout := getEnv(prefix+"DB_WRITE_TIMEOUT", "60s")
+
+	switch dbType {
+	case "mysql":
+		return fmt.Sprintf(
+			"%s:%s@tcp(%s:%s)/%s?tls=true&parseTime=true&timeout=%s&readTimeout=%s&writeTimeout=%s",
```

Comment:
if `connTimeout := getEnv(prefix+"DB_CONN_TIMEOUT", "30s")` env value doesn't exist and if it default to `"30s"`, the formatted strings will show `writeTimeout=30ss` with doiuble "s". Can you fix this